src/pal/src/sharedmemory/sharedmemory.cpp

```diff
@@ -388,6 +391,32 @@ void SharedMemoryHelpers::ReleaseFileLock(int fileDescriptor)
     } while (flockResult != 0 && errno == EINTR);
 }
 
+void SharedMemoryHelpers::CopyPath(PathCharString& destination, const char *suffix, int suffixCharCount)
```

Comment:
I find this name very confusing. Just based on the name itself when reading the code that calls it, I'd assume that it copies the suffix to the destination. Could you please make the naming clearer? It seems that e.g. BuildSharedPath or BuildSharedFilesPath could be reasonable.